release/pipelines/windows-efi-installer/README.md

```diff
@@ -111,12 +118,19 @@ spec:
         -   name: type
             value: artifact
         -   name: kind
-            value: task
+            value: pipeline
         -   name: name
             value: windows-efi-installer
         -   name: version
             value: v0.19.0
         resolver: hub
+    taskRunSpecs:
```

Comment:
Thanks @ksimon1 for finding this issue!
Why is the explicit  securityContext setting required, and could this be set via the UI?